### ISSUE TITLE
README.md

### ISSUE THREAD

**Issue by Disinterpreter** [qaisjp/mta-amx#51]
Current README.md is **too old.**
The large part of the file is invalid for our time.

I suggest to port it in google docs and make a draft from it.

Our goals in writing of README.md:
- [x] Main page
- [x] Installation
- [x] Pawn-Lua interaction
- [x] Limitations

**Comment by qaisjp** [comment]
My suggestion is that we move **some of it** to GitHub wiki: https://github.com/multitheftauto/amx/wiki

Especially the scripting docs (New Pawn scripting functions, New Lua scripting functions, New MTA events)

**Comment by Disinterpreter** [comment]
https://github.com/multitheftauto/amx/wiki/README.md-(draft)

At the moment it looks ugly. But if someone wants to support and make it beautiful, welcome!

**Comment by qaisjp** [comment]
> ~~SA-MP style rcon commands are available from the server console and the ingame console.~~

Don't these work?

**Comment by Disinterpreter** [comment]
> 
> 
> > ~SA-MP style rcon commands are available from the server console and the ingame console.~
> 
> Don't these work?

No. It works. But I think for modern samp scripters this info is useless.

**Comment by Disinterpreter** [comment]
https://github.com/multitheftauto/amx/wiki/INSTALLATION

**Comment by Disinterpreter** [comment]
I updated first post.

**Comment by NexiusTailer** [comment]
Solved by adding all remained wiki pages:
- [Pawn-Lua interaction](https://github.com/multitheftauto/amx/wiki/INTERACTION.md)
- [Limitations](https://github.com/multitheftauto/amx/wiki/LIMITATIONS.md)
